sdk/build.gradle

```diff
@@ -48,15 +48,15 @@ subprojects {
                 proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
                 buildConfigField "String", "KLAVIYO_SERVER_URL", "\"${serverTarget}\""
                 buildConfigField "String", "KLAVIYO_API_REVISION", "\"${apiRevision}\""
-                buildConfigField "String", "VERSION", "\"${versionFor(project, "version.klaviyo.versionName")}\""
-                buildConfigField "String", "NAME", "\"${klaviyoAndroidSDKName}\""
+                buildConfigField "String", "VERSION", "\"${readXmlValue('src/main/res/values/strings.xml','klaviyo_sdk_version_override', project(':sdk:core'))}\""
```

Comment:
aren't we just going to read this from the resource file at runtime? doesn't need to be a build field anymore?

Reply:
Good callout, I can remove these